features/product/viewing_products/seeing_products_lowest_price_before_the_discount.feature

```diff
@@ -16,94 +16,94 @@ Feature: Seeing the product's lowest price from 30 days before the discount
+        And this product's price changed to "$30.00"
         When I check this product's details
-        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
+        Then I should see "$21.00" as its lowest price before the discount
 
     @todo
     Scenario: Seeing the lowest price information on a product that has a discount, over a month passes, and then seeing the discount change
         Given it is "2023-03-14" now
         And the store has a product "Wyborowa Vodka" priced at "$42.00"
-        And this product's price was discounted to "$21.00"
+        And this product's price changed to "$21.00" and original price changed to "$42.00"
         And it is "2023-04-15" now
-        And this product's discounted price changed to "$33.00"
+        And this product's price changed to "$33.00"
         When I check this product's details
-        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
+        Then I should see "$21.00" as its lowest price before the discount
 
     @todo
     Scenario: Seeing the lowest price information on a product that has a discount and the one of the previous promotions ended over 30 days ago
         Given it is "2022-12-01" now
         And the store has a product "Wyborowa Vodka" priced at "$42.00"
-        And this product's price was discounted to "$21.00"
+        And this product's price changed to "$21.00" and original price changed to "$42.00"
         And it is "2023-01-01" now
-        And this product's price was changed to "$39.00"
+        And this product's price changed to "$39.00"
         And it is "2023-02-15" now
-        And this product's discounted price changed to "$33.00"
+        And this product's price changed to "$33.00"
         When I check this product's details
-        Then I should see "$39.00" as this product's lowest price from 30 days before the discount
+        Then I should see "$39.00" as its lowest price before the discount
 
     @todo
     Scenario: Not seeing the lowest price information on a product that has a discount and the previous promotion ended over 30 days ago
         Given it is "2022-12-01" now
         And the store has a product "Wyborowa Vodka" priced at "$42.00"
-        And this product's price was discounted to "$21.00"
+        And this product's price changed to "$21.00" and original price changed to "$42.00"
         And it is "2023-01-01" now
-        And this product's discount was removed
+        And this product's price changed to "$42.00" and original price was removed
         And it is "2023-02-15" now
-        And this product's price was discounted to "$33.00"
+        And this product's price changed to "$33.00" and original price changed to "$42.00"
         When I check this product's details
-        Then I should not see information about its lowest price
+        Then I should see "$42.00" as its lowest price before the discount
```

Comment:
:+1: but the scenario title has to be changed. And we need a new scenario that realizes it.